warrior/Framework/Utils/data_Utils.py

```diff
@@ -901,10 +899,15 @@ def validate():
     keys = key.split('.')
     value = get_object_from_datarepository(keys[0])
     for k in keys[1:]:
+        if k not in value:
+            print_warning("key {} not present in data repository\n".format(key))
         value = value[k]
     if not value:
         err_msg += "key {} not present in data repository\n".format(key)
         result = "ERROR"
+    if value:
```

Comment:
Hi @sugan19,

Since we have the ```if not value``` block on line 905, can we just have an ```else``` instead of ```if value``` on line 908?

Thanks.

Reply:
@sanikakulkarni  Addressed you comments.